### PR TITLE
Fixing Dark Mode (Desktop) and Unifying the Extension

### LINKED ISSUE #420: Grouped Items are Unreadable in Dark Mode
Thanks for fixing the previous issue with grouping so quickly! It's working great now, except in dark mode. The dropdowns from a grouped box are grey and therefore, unreadable in dark mode.

image

### PR DESCRIPTION
**Pull request by mindlessroman**
This is a big PR! It aims to fix #420, #409. Will put in a separate PR for Mobile view (#356), just to break these up into as "small" of PRs as I can.

While going through the code making sure things looked readable in both dark and light themes, there were some modal and dialog boxes that needed a bit of visual TLC. These areas got theme-testing as well as some visual information organizing.

(FWIW, I used some pretend profile pictures in these screenshots, sourced from Adobe Stock. Thanks Adobe!)

Now for some screenshot demos! _Please note that there are some very minor tweaks from what is shown in the screenshot._

---

Main View, Light Mode
image

Main View, Dark Mode
image

Create a New Retrospective Dialog, Light Mode
image

Create a New Retrospective Dialog, Dark Mode
image

Edit Retrospective Dialog, Light Mode
image

Edit Retrospective Dialog, Dark Mode
image

Seeing Grouped Items in the "Group Phase", Light Mode
image

Seeing Grouped Items in the "Group Phase", Dark Mode
image

Seeing Action Items in "Act Phase", Light Mode
image

Seeing Action Items in "Act Phase", Dark Mode
image

Focus Mode showing Grouped Items and Action Items beneath the highlight item, Light Mode
image

Focus Mode showing Grouped Items and Action Items beneath the highlight item, Dark Mode
image

Feedback Action Menu Ellipsis, Light Mode
image

Feedback Action Menu Ellipsis, Dark Mode
image

Delete Retrospective Board, Light Mode
image

Delete Retrospective Board, Dark Mode
image

Prime Directive Dialog, Light Mode
image

Prime Directive Dialog, Dark Mode
image

Retrospective Summary without Team Assessment, Light Mode
image

Retrospective Summary without Team Assessment, Dark Mode
image

Team Assessment questionnaire, Light Mode
image

Team Assessment questionnaire, Dark Mode
image

Retrospective Summary with Team Assessment, Light mode
image

Retrospective Summary with Team Assessment, Dark Mode
image

History View, Light mode
image

History View, Dark mode
image

History View with a Retrospective Summary Expanded, Light Mode
image

History View with a Retrospective Summary Expanded, Dark mode
image

### CODE REVIEW

**Comment by mindlessroman** [comment]
And would address #409 as well.

**Review by polatengin — APPROVED** [review]
🙌🍾🎈

File: RetrospectiveExtension.Frontend/components/feedbackBoardContainer.tsx [hunk 1]
```diff
@@ -1194,7 +1200,7 @@ class FeedbackBoardContainer extends React.Component<FeedbackBoardContainerProps
                             items: this.boardActionContexualMenuItems,
                           }}
                         >
-                          <span className="ms-Button-icon"><i className="fas fa-ellipsis-h"></i></span>&nbsp;
+                          <span className="ms-Button-icon"><i className="fa-solid fa-grip"></i></span>&nbsp;
```
Comment (evancharlton) [inline RetrospectiveExtension.Frontend/components/feedbackBoardContainer.tsx:1203]: **question**: Is there a reason that this was changed to "grip"?

I thought that the `...` pattern was well-established, and the `:::` icon was for drag-and-drop affordances.

Just curious :)

Comment (evancharlton) [reply RetrospectiveExtension.Frontend/components/feedbackBoardContainer.tsx:1203]: If this was inadvertent, I've created #456 to fix it

**Pull request merged**